test/lint.js

```diff
@@ -237,6 +237,9 @@ describe('linting whole program', function () {
           <emu-clause id="i7">
             <h1>Function ( _p1_, _p2_, &hellip; , _pn_, _body_ )</h1>
           </emu-clause>
+          <emu-clause id="foo">
+            <h1>Example ( _a_, <del>_b_</del><ins>_c_</ins> )</h1>
```

Comment:
Can you add a test showing that errors still work as well?

Reply:
I added one checking that `Example ( _a_ [ , <del>_b_</del> ] )` gives an error — is that the sort of test you meant?

Reply:
You should probably also add a test that `Example ( _a_ <del>[ , _b_ ]</del> )` and `Example ( _a_ <ins>[ , _b_ ]</ins> )` work correctly.

Reply:
Good suggestion; by doing that I found out that leading and trailing spaces inside `<ins>` and `<del>` tags should be allowed, so I've added another commit changing that.